docker/webui/nginx.conf

```diff
@@ -0,0 +1,24 @@
+worker_processes  1;
+
+events {
+    worker_connections  1024;
+}
+
+http {
+    server {
+        listen 80;
+        server_name  localhost;
```

Comment:
can/should this file be formatted using 2 spaces?

Reply:
tbh this is a pretty old known to be working `nginx.conf` I had so I didn't give it any thought. I'll check what's the usual identation and clean it up 👍